src/runner.rs

```diff
@@ -1,21 +1,41 @@
-use crate::constants::EXIT_FAILURE;
+use crate::clients::get_quarantine_bulk_test_status;
+use crate::constants::{EXIT_FAILURE, EXIT_SUCCESS};
 use crate::scanner::{BundleRepo, FileSet, FileSetCounter};
-use crate::types::{RunResult, Test};
+use crate::types::QuarantineBulkTestStatus;
+use crate::types::{QuarantineRunResult, RunResult, Test};
 use junit_parser;
 use std::fs::metadata;
 use std::process::Command;
 use std::process::Stdio;
 use std::time::SystemTime;
+use tokio_retry::strategy::ExponentialBackoff;
+use tokio_retry::Retry;
```

Comment:
IMO, it's much nicer to group imports

```suggestion
use crate::{
    clients::get_quarantine_bulk_test_status,
    constants::{EXIT_FAILURE, EXIT_SUCCESS},
    scanner::{BundleRepo, FileSet, FileSetCounter},
    types::{QuarantineBulkTestStatus, QuarantineRunResult, RunResult, Test},
};
use anyhow::Ok;
use junit_parser;
use std::{
    fs::metadata,
    process::{Command, Stdio},
    time::SystemTime,
};
use tokio_retry::{strategy::ExponentialBackoff, Retry};
```